backend/jest.config.ts

```diff
@@ -0,0 +1,6 @@
+import { pathsToModuleNameMapper } from 'ts-jest/utils'
+import { compilerOptions } from './tsconfig.json'
+
+module.exports = {
+  preset: 'ts-jest',
+}
```

Comment:
Turned this into a TS module to accommodate typing issues.

Reply:
Okay